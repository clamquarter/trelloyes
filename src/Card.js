import React from 'react'
import './Card.css'

class Card extends React.Component{
render(){
  return   (<div className='Card'>
  <button type='button'>
    Delete
  </button>
  <h3>Title{this.props.title}</h3>
  <p>Testing Card{this.props.content}</p>
    </div>)
}
}


export default Card