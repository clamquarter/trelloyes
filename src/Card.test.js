import React from 'react'
import ReactDOM from 'react-dom'
import Card from './Card'
import renderer from 'react-test-renderer'

it('renders without crash', () => {
    const div = document.createElement('div')

    ReactDOM.render(<Card />, div)
    ReactDOM.unmountComponentAtNode(div)
})

it('renders the UI as expected', () => {
    const tree = renderer
    .create(<Card title='james bond' content='lorem ipsum hipster' />)
    .toJSON()
    expect(tree).toMatchSnapshot()

})