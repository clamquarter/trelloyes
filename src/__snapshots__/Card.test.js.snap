// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    Delete
  </button>
  <h3>
    Title
    james bond
  </h3>
  <p>
    Testing Card
    lorem ipsum hipster
  </p>
</div>
`;
